### DISCUSSION TITLE
What versions of Windows can the GLPI-Agent v1.4 be installed on ?

### DISCUSSION THREAD

**Discussion by sebma** [glpi-project/glpi-agent#292] [Q&A]
Hi,

Can the GLPI-Agent v1.4 be installed on previous versions of Windows (XP, 7, 8, 8.1, 2003, 2008, 2012, 2016) ?

Or in other words : Do you have a list of compatible Windows versions ?

**Comment by g-bougard** [comment]
Hi @sebma 
we don't have an official list and we don't want to provide support for too old versions.
Anyway, as GLPI-Agent is a fork of FI Agent which was running on XP and newer, you can expect GLPI-Agent can still run on XP.
By the way, GLPI-Agent is provided as MSI packaging vs FI which was provided as EXE installer. So to use the MSI, the targeted system should support MSI too (I don't remember exactly since when MSI packaging is supported, XP SP2 ?).
But, you still have the portable version which should probably run on XP out of the box as perl should support it.
Then if you find an issue for XP, it won't probably merged upstream unless you provide the code yourself in a PR ;-)
So just try it.
